src/scanners/rdf.js

```diff
@@ -24,7 +24,7 @@ export default class RDFScanner extends BaseScanner {
         },
       }).parseFromString(this.contents, 'text/xml');
 
-      resolve(xmlDoc);
+      resolve(xmlDoc, null, {namespace: this.namespace});
```

Comment:
Seems I already partially set this up; in `BaseScanner` calls to rule functions are made like this: https://github.com/mozilla/addons-validator/blob/master/src/scanners/base.js#L31

So really we want to be setting `this.options.namespace = this.namespace`;

We should define `this.options = {}` by default in `BaseScanner` too.